TFNetworkRecLayer.py

```diff
@@ -4457,13 +4475,7 @@ def get_out_data_from_opts(cls, name, sources, target, network,
       del shape[out_data.batch_dim_axis]
       out_data = Data(name="%s_output" % name, shape=shape, sparse=True, dim=out_data.dim)
     if search:
-      from TFUtil import SearchBeam
-      search_dep = NotSpecified
-      if sources and sources[0] and not sources[0].output.undefined:
-        search_dep = sources[0].output.beam
-      out_data.beam = SearchBeam(
-        beam_size=beam_size, dependency=search_dep,
-        name="%s%s" % (network.get_absolute_name_prefix(), name))
+      out_data.beam = cls._create_search_beam(name, beam_size, sources, network)
```

Comment:
You should always use named kwargs if possible, if you have more than 1 param or there can be any ambiguity from the function name about the params.

Reply:
So, is this resolved? (If you resolve sth, and the old code gets outdated, I cannot see the new code, so I don't directly see it. Please click yourself on "resolve".)